docs/src/content/docs/commands/GETRANGE.MD

```diff
@@ -0,0 +1,107 @@
+---
+title: GETRANGE
+description: The `GETRANGE` command in DiceDB is used get a substring of string provided the start and end.
+---
+The `GETRANGE` command in DiceDB is used get a substring of string provided the start and end.
```

Comment:
Shall we draft it as?
`The GETRANGE command in DiceDB is used to get a substring of a string, provided the start and end indices`